tests/vector/test_spaces.py

```diff
@@ -95,6 +95,32 @@ def test_batch_space(space, expected_batch_space_4):
     assert batch_space_4 == expected_batch_space_4
 
 
+def test_batch_space_seed():
+    for space in [
```

Comment:
This seems unrelated to the rest of the test, should probably be a different test, and use a parametrized `space` argument.

Reply:
Thanks for the comments

I didn't use a parameterized pytest because this test is not possible for all of the spaces (Discrete and MultiDiscrete) as there randomness is different then we can't check the values produced are the same. 
So to test all of the Spaces in one functions, this was the only way.

I agree, if we could find a way that the batched spaces produced the same random values as the original space then we should change to parameterized. Does that make sense or do you think we should still change it?

Reply:
I'd parametrize with only a few spaces first then. For loops like these don't belong in tests imo.

Reply:
> we can't check the values produced are the same.

See other thread for a more detailed response

Reply:
This can be marked as resolved.